.github/workflows/integration_tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Integration Testing
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    environment:
+      name: ci_testing
+    strategy:
+      fail-fast: true
+      max-parallel: 3
+      matrix:
+        python-version: [ "3.11"] # "3.10", "3.12"]
+        dbt-version: ["1.7.0"] # "1.6.0", , "1.8.0b1"]
```

Comment:
We used to run ci only on prereleases - might be worth doing:
* Current stable release (`pip install dbt-core dbt-snowflake`).
* Latest prerelease (`pip install dbt-core dbt-snowflake --pre`).

But maybe something to think about for later :)

Reply:
100% will have it again eventually. currently blocked on this by https://github.com/dbt-labs/dbt-external-tables/issues/271